spec/features/projects/projects_index_spec.rb

```diff
@@ -280,51 +280,51 @@ def load_and_open_filters(user)
         expect(error_container["innerHTML"]).to include error_html
       end
     end
-  end
 
-  describe "project attributes visibility restrictions" do
-    let(:user) do
-      create(:user,
-             member_with_roles: {
-               development_project => create(:existing_project_role, permissions:),
-               project => create(:existing_project_role)
-             })
-    end
+    context "for project attributes", with_ee: %i[custom_fields_in_projects_list] do
```

Comment:
I thought the enterprise action `custom_fields_in_projects_list` was removed?
https://github.com/opf/openproject/commit/792def7b90b

Reply:
We still seem to have the enterprise check with `EnterpriseToken.allows_to?(:custom_fields_in_projects_list)` in the filter component:

https://github.com/opf/openproject/blob/caf494516c086786bd3b2737eba89a9a915ed0eb/app/components/projects/projects_filters_component.html.erb#L108-L117

Should I remove that ?